src/Core/Services/TypeHelper.cs

```diff
@@ -2,120 +2,202 @@
 // Licensed under the MIT License.
 
 using System.Data;
+using System.Net;
 using Azure.DataApiBuilder.Core.Services.OpenAPI;
+using Azure.DataApiBuilder.Service.Exceptions;
 
-namespace Azure.DataApiBuilder.Core.Services
+namespace Azure.DataApiBuilder.Core.Services;
+
+/// <summary>
+/// Type mapping helpers to convert between SQL Server types, .NET Framework types, and Json value types.
+/// </summary>
+/// <seealso cref="https://learn.microsoft.com/dotnet/framework/data/adonet/sql-server-data-type-mappings"/>
+public static class TypeHelper
 {
     /// <summary>
-    /// Helper class used to resolve CLR Type to the associated DbType or JsonDataType
+    /// Maps .NET Framework types to DbType enum
+    /// Unnecessary to add nullable types because
```

Comment:
nit: incomplete reasoning? why not adding nullable types?

Reply:
updated comment to apply to correct dictionary. Not this one (can be a follow up PR since changing this one would be a separate concern.)
also added unit test and explanation for not adding nullable types to the dictionary.